cpp/src/filling/legacy/tile.cu

```diff
@@ -32,6 +34,8 @@ namespace cudf {
 namespace detail {
 
 cudf::table tile(const cudf::table &in, gdf_size_type count, 
+                 rmm::mr::device_memory_resource* mr =
```

Comment:
Why did this legacy API need to be modified?

Reply:
Accidental. Thank you for catching it.